### DISCUSSION TITLE
error on chooser, crazyflie_tools not found

### DISCUSSION THREAD

**Discussion by frulloa** [USC-ACTLab/crazyswarm#731] [Q&A] [answered]
Hi, everybody. 
Please, your kindly help. I am stuck in this stuff for days.
What can I do?, please your help

I already set up the crazyswarms following the configuration guidelines https://crazyswarm.readthedocs.io/en/latest/configuration.html
Unfortunately, when I arrive to the part of chooser.py if I press "battery" I have this error: [rospack] Error: package 'crazyflie_tools' not found

![Screenshot from 2023-03-22 12-26-38](https://user-images.githubusercontent.com/78211872/226890575-6bd7034b-a753-4448-bc37-30c4829e38a4.png)

The setup of the crazyflies.yaml and  allCrazyflies.yaml are like this:
![Screenshot from 2023-03-22 12-27-55](https://user-images.githubusercontent.com/78211872/226890823-2c7201f3-02f0-47d3-b8ef-354e9259a9a0.png)

Additionally, the hover_swarm.launch is as follows:
I changed motion_capture_type: "optitrack_closed_source" 
object_tracking_type: "motionCapture"
motion_capture_host_name: "192.168.1.3"

![Screenshot from 2023-03-22 12-32-56](https://user-images.githubusercontent.com/78211872/226891844-a3163a6d-6b82-4488-a28d-6bfcbb89ceed.png)

**Comment by jpreiss** [comment, marked as answer]
Did you source `devel/setup.bash`?

**Reply by frulloa** [reply]
Hi, thanks for your response. Yes, when I run the following commands the RViz open
![image](https://user-images.githubusercontent.com/78211872/226742964-0505a4f6-7250-481f-8dc8-492b8b11cc31.png)
